src/main/java/Deadline.java

```diff
@@ -0,0 +1,64 @@
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+public class Deadline extends Task {
+    private final LocalDate DeadlineDate;
+    private final LocalTime DeadlineTime;
+
+    /**
+     * Constructor of Deadline class.
+     *
+     * @param name name of deadline.
+     * @param isDone deadline status.
+     * @param DeadlineDate date of deadline
+     * @param DeadlineTime time of deadline
+     *
+     */
+    Deadline(String name, Boolean isDone
+            , LocalDate DeadlineDate, LocalTime DeadlineTime) {
+        super(name,isDone);
+        this.DeadlineDate = DeadlineDate;
+        this.DeadlineTime = DeadlineTime;
+    }
+
+    /**
+     * Get deadline date
+     *
+     * @return deadline date.
+     */
+
+    public LocalDate getDeadlineDate(){
+        return this.DeadlineDate;
+    }
+
+    /**
+     * Get deadline time
+     *
+     * @return deadline time.
+     */
+
+    public LocalTime getDeadlineTime() {
+        return DeadlineTime;
+    }
+
+    /**
+     * Mark task as done
+     *
+     * @return new task, marked as done.
+     */
+    @Override
+    Deadline finish(){
```

Comment:
Remember to have a spacing between the method name and the open curly bracket :)
e.g finish() { instead of finish(){